lib/Varien/Autoload.php

```diff
@@ -66,7 +66,7 @@ public function autoload($class)
         /** @see https://stackoverflow.com/a/5504486/716029 */
         $found = stream_resolve_include_path($path);
         if ($found !== FALSE) {
```

Comment:
```suggestion
       if ($found !== false) {
```

To avoid php-cs-fixer error.